atlas/export/jsonl.py

```
@@ -0,0 +1,485 @@
+        logger.warning("Session %s not found; skipping", session_id)
+        return None
+
+    detailed = session_row
+    if "plan" not in detailed or detailed["plan"] is None:
+        fetched = await store.fetch_session(session_id)
+        if fetched is None:
+            logger.warning("Session %s disappeared during export; skipping", session_id)
+            return None
+        detailed = fetched
+
+    plan_model, plan_dict, plan_lookup = _coerce_plan(detailed.get("plan"))
```

Ruff (0.13.3)

Unpacked variable `plan_model` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)